container_pipeline/start_build.py

```diff
@@ -0,0 +1,75 @@
+import json
+import logging
+import os
+import sys
+
+import container_pipeline.utils as utils
+from container_pipeline.lib import settings
+from container_pipeline.lib.log import load_logger
+from container_pipeline.lib.openshift import Openshift, OpenshiftError
+from container_pipeline.lib.queue import JobQueue
+
+
+def create_project(appid, jobid, repo_url, repo_branch, repo_build_path, target_file, notify_email, desired_tag, depends_on, test_tag):
+    job_name = utils.get_job_name({
+        'appid': appid, 'jobid': jobid, 'desired_tag': desired_tag})
+    project = utils.get_job_hash(job_name)
+    is_openshift_good = True
+
+    openshift = Openshift(logger=logger)
+    try:
```

Comment:
Redundant ``try...except`` blocks. Prefer raising exceptions and exiting the function as early as possible.